### PR TITLE
updated layout style, add list design :wip

### PR DESCRIPTION
**Pull request by hitankar-freshworks**

### CODE REVIEW

File: src/components/ListItem.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col'
+import Badge from 'react-bootstrap/Badge';
+
+const ListItem = () => (
+	<Card className="mb-3">
+		<Card.Header as="h5">
+			Sample Component
+		<div className="float-right">
```
Comment (ghost) [inline src/components/ListItem.js]: Is there any other selector we can use other than float?

File: src/components/ListItem.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (ghost) [inline src/components/ListItem.js:1]: Indentation seems off in this file. Is this just GitHub?

File: src/components/ListItem.js (hunk 1 above)
Comment (hitankar-freshworks) [reply src/components/ListItem.js]: I feel it should be fine in this case. We can use flex alignment improvement later

File: src/components/Header.js [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
-      <Link to={Routes.Intro} className="header__logo">
-        <img src={Logo} alt="Logo" />
-      </Link>
-      <Link to={Routes.Intro} className="header__nav-item">Common Components Catalogue</Link>
-      <Link to={Routes.Intro} className="header__nav-item">About</Link>
-      <Link to={Routes.ListView} className="header__nav-item">Catalogue</Link>
-    </Row>
+    <Nav>
+      <Nav.Item>
+        <Nav.Link className="header__nav-item" className="header__logo" alt="Logo" as={Link} to={Routes.Intro}><img src={Logo}/></Nav.Link>
```
Comment (ashtonmeuser) [inline src/components/Header.js]: `className` is set twice here.

**Review by ashtonmeuser — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**